src/SFA.DAS.AANHub.Application.UnitTests/Queries/WhenRequestingCalendars.cs

```diff
@@ -56,12 +68,10 @@ public class WhenRequestingCalendars
             var calendars = new List<Calendar> { calendar };
             var calendarPermissions = new List<CalendarPermission> { calendarPermission };
 
-            context.Calendars.AddRange(calendars);
-            context.CalendarPermissions.AddRange(calendarPermissions);
-            await context.SaveChangesAsync();
-
-            var result = await handler.Handle(query, CancellationToken.None);
+            _calendarsReadRepository.Setup(m => m.GetAllCalendars()).ReturnsAsync(calendars);
+            _calendarsPermissionsReadRepository.Setup(m => m.GetAllCalendarsPermissions()).ReturnsAsync(calendarPermissions);
 
+            var result = await _handler.Handle(query, CancellationToken.None);
 
             var getCalendarsResultItems = result.ToList();
```

Comment:
Where is the number 2345 coming from in the code below?

Reply:
Here: https://github.com/SkillsFundingAgency/das-aan-hub-api/blob/9bbd5946f94207142bd0a4419cf046b1c3ca5389/src/SFA.DAS.AANHub.Application.UnitTests/Queries/WhenRequestingCalendars.cs#L63

It's just a hardcoded value added to a dummy CalendarPermission to ensure that a correctly linked calendar gets the expected permission id